make/tools.mk

```diff
@@ -511,6 +511,101 @@ endif
 zip_clean:
 	$(V1) [ ! -d "$(ZIP_DIR)" ] || $(RM) -rf $(ZIP_DIR)
 
+
+# Google depot-tools, used to build breakpad on Mac/Linux
+DEPOT_TOOLS_REPO := https://chromium.googlesource.com/chromium/tools/depot_tools.git
+DEPOT_TOOLS_REV := master
+DEPOT_TOOLS_DIR := $(TOOLS_DIR)/depot-tools
+
+.PHONY: depot_tools_install depot_tools_clean
+depot_tools_install: | $(TOOLS_DIR)
+	$(V0) @echo " DOWNLOAD     $(DEPOT_TOOLS_REPO)"
+	$(V1) ( \
+		if [ ! -d "$(DEPOT_TOOLS_DIR)" ] ; then \
+			mkdir -p "$(DEPOT_TOOLS_DIR)" ; \
+			cd "$(DEPOT_TOOLS_DIR)" ; \
+			git init -q ; \
+			git remote add origin $(DEPOT_TOOLS_REPO) ; \
+		fi ; \
+		cd "$(DEPOT_TOOLS_DIR)" ; \
+		git fetch -q --depth=1 origin $(DEPOT_TOOLS_REV) ; \
+		git checkout -q -f FETCH_HEAD ; \
+		git clean -q -f -d -x ; \
+	)
+
+depot_tools_clean:
+	$(V0) @echo " CLEAN        $(DEPOT_TOOLS_DIR)"
+	$(V1) [ ! -d "$(DEPOT_TOOLS_DIR)" ] || $(RM) -rf $(DEPOT_TOOLS_DIR)
+
+# Google breakpad
+BREAKPAD_REPO := https://github.com/d-ronin/breakpad.git
+BREAKPAD_REV := 20160909
+BREAKPAD_DIR := $(TOOLS_DIR)/breakpad
+BREAKPAD_BUILD_DIR := $(DL_DIR)/breakpad
+
+.PHONY: breakpad_install breakpad_clean breakpad_build_clean
+
+ifndef WINDOWS
+
+breakpad_install: | $(DL_DIR) $(TOOLS_DIR)
+breakpad_install: | breakpad_clean
+	$(V0) @echo " DOWNLOAD     $(BREAKPAD_REPO) @ $(BREAKPAD_REV)"
+	$(V1) mkdir -p "$(BREAKPAD_BUILD_DIR)"
+
+	$(V1) ( \
+		cd $(BREAKPAD_BUILD_DIR) ; \
+		export PATH="$(DEPOT_TOOLS_DIR):$(PATH)" ; \
+		gclient config --name=src "$(BREAKPAD_REPO)" ; \
+		gclient sync --with_tags --no-history -r refs/tags/$(BREAKPAD_REV) ; \
+	)
+
+	$(V0) @echo " BUILD        $(BREAKPAD_DIR)"
+	$(V1) ( \
+		cd "$(BREAKPAD_BUILD_DIR)/src/src" ; \
+		$(MAKE) distclean ; \
```

Comment:
Can't do this until configure has run once, and need to do it before configure runs. Will have to check to see if Makefile exists.